models/src/main/protobuf/RhoTypes.proto

```diff
@@ -251,24 +237,8 @@ message EOr {
     Par p2 = 2;
 }
 
-message GBool {
-    bool b = 1;
-}
-
-message GInt {
-    sint32 i = 1;
-}
-
-message GString {
-    string s = 1;
-}
-
-message GUri {
-    string u = 1;
-}
-
 // These should only occur as the program is being evaluated. There is no way in
 // the grammar to construct them.
 message GPrivate {
-    string p = 1;
+    string id = 1;
```

Comment:
For deterministic serialization, these might need to be bytes.
Fortunately they're compatible either way, so leave them for now.